params.ini

```diff
@@ -361,4 +361,4 @@ uniq_weapon_titan_revenge=1
 uniq_weapon_tomb_reaver=1
 uniq_weapon_warshrike=1
 uniq_weapon_windforce=1
-uniq_weapon_wizardspike=1
+uniq_weapon_wizardspike=1
```

Comment:
Dont remove new lines at end of files all the time : )

Reply:
https://github.com/aeon0/botty/blob/master/params.ini

I removed because this on on main branch didn't had a new line at end of file